Watch Extension/State Management/SavedFyiDialogReducer.swift

```diff
@@ -5,11 +5,13 @@ enum SavedFyiDialogAction {
     case hide
 }
 
-func savedFyiDialogReducer(showSavedFyiDialog: inout Bool, action: SavedFyiDialogAction) {
+func savedFyiDialogReducer(showSavedFyiDialog: inout Bool, action: SavedFyiDialogAction) -> [Effect<SavedFyiDialogAction>] {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/995ebb3b78eb3cd87dba718e07e1f295f49795f1/Watch Extension/State Management/SavedFyiDialogReducer.swift#L8"></span>Line should be 120 characters or less: currently 124 characters<br />
<code>line_length</code> <code>SavedFyiDialogReducer.swift:8</code></td>
    </tr>
  </tbody>
</table>